### PR TITLE
feat(*): add ROADMAP.md and CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by kdada**
**What this PR does / why we need it**:

Add ROADMAP.md and CONTRIBUTING.md

**Which issue(s) this PR fixes** *(optional, close the issue(s) when PR gets merged)*:

Fixes #

**Special notes for your reviewer**:

/cc @ddysher 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  - Operator ([#5](https://github.com/caicloud/nirvana/issues/5))
+- Client Generation
+  - go ([#41](https://github.com/caicloud/nirvana/issues/41))
+  - graphql
+- Refine Error ([#72](https://github.com/caicloud/nirvana/issues/72) [#200](https://github.com/caicloud/nirvana/issues/200))
+- Nirvana cli [#203](https://github.com/caicloud/nirvana/issues/203)
+  - Project structure
+- Test coverage 60% ([#67](https://github.com/caicloud/nirvana/issues/67))
+- Performance ([#148](https://github.com/caicloud/nirvana/issues/148))
+- Monitoring
```
Comment (ddysher) [inline ROADMAP.md]: add more details or link to an issue?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+6. Merge your pull request.
+
+#### Fix typos, commments or submit docs
+1. Modify and push to your own repository.
+2. Create a pull request to Nirvana.
+3. Code review and modify your codes if it's necessary.
+4. Merge your pull request.
+
+### Code review
+
```
Comment (ddysher) [inline CONTRIBUTING.md]: an automatic tool

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+After that, you can immediately clone `https://github.com/<your-account-name>/nirvana` to your local path `$GOPATH/src/github.com/caicloud/nirvana`.  
+There are some environment requirements:  
+
+- [Latest golang](https://golang.org/dl/)
+- [Latest dep](https://github.com/golang/dep/)
+- [Latest gometalinter](https://github.com/alecthomas/gometalinter)
+
+### Find something you are interested in
+
+Nirvana is in rapid development, many of things are not complete. There are many components that you can dig into:
```
Comment (ddysher) [inline CONTRIBUTING.md]: s/many of things/many things

**Comment by kdada** [comment]
@ddysher 
PTAL

**Pull request merged**